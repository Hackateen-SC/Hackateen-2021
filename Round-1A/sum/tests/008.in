57
524
420
152
399
658
597
163
310
119
146
425
271
80
949
875
488
519
638
381
656
395
770
749
92
577
676
930
348
761
950
309
976
480
281
125
718
834
664
280
319
590
957
42
410
890
378
809
117
723
937
889
293
989
967
421
863
708
