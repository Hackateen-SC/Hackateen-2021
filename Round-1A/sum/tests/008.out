18537607
4868589
18407741
1857720
17423934
9524973
11606646
7370023
0
7768587
11407126
11751175
11628834
1290129
11535022
13911391
16498088
0
10603800
16451645
7867636
5590559
6182756
2723697
14687235
5513976
0
15091509
19941035
15599368
9627198
4270373
6810681
12540305
0
6186969
260897
1037775
18441625
15053395
12178546
47128
2950660
0
14468339
6350498
11198791
12901290
6573066
0
5529456
2405420
12907568
19785590
7081584
15397963
14949524
